pkg/server/handler/metrics/handler.go

```diff
@@ -25,13 +27,59 @@ func New(c Config) *Handler {
 	if c.Log == nil {
 		tracer.Panic(tracer.Mask(fmt.Errorf("%T.Log must not be empty", c)))
 	}
-	if c.Reg == nil {
-		tracer.Panic(tracer.Mask(fmt.Errorf("%T.Reg must not be empty", c)))
+	if c.Met == nil {
+		tracer.Panic(tracer.Mask(fmt.Errorf("%T.Met must not be empty", c)))
+	}
+
+	cou := map[string]recorder.Interface{}
+
+	{
+		nam := "teams_bridge_total"
+		cou[nam] = recorder.NewCounter(recorder.CounterConfig{
+			Des: "the amount of total bridge transactions",
+			Lab: map[string][]string{
+				"success": {"true", "false"},
+			},
+			Met: c.Met,
+			Nam: nam,
+		})
+	}
```

Comment:
This is how we would whitelist a metric inside the desired handler. It's not pretty but this configuration should in general not change frequently.